site/components/Navigation/NavItemDesktop.tsx

```diff
@@ -23,7 +23,7 @@ export const NavItemDesktop: FC<Props> = ({
         <Link href={url}>
           <a
             className={cx(styles.navMain_DesktopLink, {
-              dropdown: subMenu !== undefined,
+              dropdown: !!subMenu,
```

Comment:
these url links don't have a dropdown/submenu, so I assume this and `data-active` can be removed

Reply:
yes, you're right. removed.